### PR TITLE
Disable predictive capacity alerts by default

### PR DESCRIPTION
**Pull request by schemen**
If required, these alerts can be enabled by choice on a cluster. As a baseline, the
standard non-predictive alerts are enough.

This reduces the amount of tenant data written.

## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

**Review by simu — CHANGES_REQUESTED** [review]
Please update the documentation to note that the predictive alerts are not enabled by default.

A note in https://github.com/appuio/component-openshift4-monitoring/blob/master/docs/modules/ROOT/pages/references/parameters.adoc#capacityalerts should be sufficient, I think.

**Comment by schemen** [comment]
I have updated the documentation regarding CpuCapacity and predictive alerts.

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 1]
```diff
@@ -288,6 +288,8 @@ type:: dict
 
 This parameter allows users to enable and configure alerts for capacity management.
 The capacity alerts are disabled by default and can be enabled by setting the key `capacityAlerts.enabled` to `true`.
+Predictive alerts are disabled by default and can be enabled, as exampled by `ExpectClusterCpuUsageHigh.enabled` to `true`.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc:291]:
```suggestion
Predictive alerts are disabled by default.
They can be enabled individually as shown in the example by setting field `<AlertName>.enabled` to `true`.
```

File: class/defaults.yml [hunk 2]
```diff
@@ -242,7 +242,7 @@ parameters:
         CpuCapacity:
           rules:
             ClusterCpuUsageHigh:
-              enabled: true
+              enabled: false
```
Comment (simu) [inline class/defaults.yml:245]: I don't understand why we disable this alert by default. As an operator I would want to know if my cluster has high CPU usage.

Also, this change should probably be in a different PR, this PR currently contains two unrelated changes:

1. Disable predictive alerts by default
2. Disable ClusterCpuUsageHigh by default

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 3]
```diff
@@ -288,6 +288,8 @@ type:: dict
...
+Also, `CpuCapacity` alerts are also disabled by default, only required for compute heavy clusters.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc:292]:
```suggestion
Additionally, all `CpuCapacity` alerts are disabled by default.
They're only required for compute heavy clusters.
```

I don't agree that "CpuCapacity" alerts are only required for compute heavy clusters. What's the actual reason for disabling the non-predictive CpuCapacity alert by default?

**Review by simu — APPROVED** [review]
